package/CHANGELOG

```diff
@@ -31,6 +31,8 @@ Changes
   * Maximum pinned versions in setup.py removed for python 3.6+ (PR #3139)
 
 Deprecations
+  * Deprecate structure of `results` attribute for LinearDensity, 
+    PersistenceLength and AnalysisFromFunction (Issue #3115)
```

Comment:
@PicoCentauri AnalysisFromFunction is mentioned here, but it's not included in the changes of this PR?